src/App.tsx

```diff
@@ -1,24 +1,173 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import {
+  getTodos,
+  addTodo,
+  deleteTodo,
+  completeTodo,
+} from './api/todos';
+import { Filter } from './types/Filter';
+import { ErrorTypes } from './types/ErrorTypes';
+import {
+  Header, TodosList, Footer, ErrorNotifications,
+} from './components';
 
-const USER_ID = 0;
+const USER_ID = 6335;
 
 export const App: React.FC = () => {
+  const [allTodos, setAllTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<ErrorTypes | null>(null);
+  const [filter, setFilter] = useState<Filter>(Filter.All);
+  const [title, setTitle] = useState<string>('');
+  const [isAdding, setIsAdding] = useState<boolean>(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [selectedTodoId, setSelectedTodoId] = useState(0);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(setAllTodos)
+      .catch(() => {
+        setError(ErrorTypes.Load);
+        setTimeout(() => setError(null), 3000);
```

Comment:
you can create function for this operation & you will can easily put this function to another down component
```suggestion
        setTimeout(cleareError, 3000);
```